config/rbac/kustomization.yaml

```diff
@@ -1,8 +1,11 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-commonLabels:
-  app.kubernetes.io/name: keda-operator
-  app.kubernetes.io/part-of: keda-operator
+
+labels:
+  - pairs:
+      app.kubernetes.io/name: keda-operator
+      app.kubernetes.io/part-of: keda-operator
+    includeSelectors: true
```

Comment:
Why is this change needed?

Reply:
This change maintains feature parity with the current implementation of `commonLabels`. When you use `commonLabels,` it adds label selectors to all Kubernetes resources. This change addresses the deprecation warning without changing functionality/resulting manifests.

Reply:
Thanks!